### ISSUE TITLE
feature request: option to deploy libvirt as flatpak addon

### ISSUE THREAD

**Issue by wackywobblewave** [flathub/org.virt_manager.virt-manager#7]
_Short disclaimer, I already mentioned this in https://github.com/flathub/org.virt_manager.virt-manager/issues/3, but since it was rather off topic there, I decided to create a separate issue._

**Why?**
- Easier for beginners
- There is no convenient way to install libvirt for users of immutable distributions like aeon, gnome os, silverblue etc.

**two approaches that come to my mind**:
a)
(IMO the better one)
**Provide libvirt as a flatpak extension: https://docs.flatpak.org/en/latest/extension.html** 
This has the advantage that it is still a pure virt-manager client for ssh / libvirt on the host, but it can be easily extended as an flatpak extension / Add-on with libvirt via the local software store like gnome software or kde's discover.

b)
**Adding libvirt to the main manifest**
This would have the downside of possible bloatware for some users.

**Comment by travier** [comment]
The current path I'm working on is to deploy it via a sysext: https://github.com/travier/fedora-sysexts

**Comment by TheRsKing** [comment]
That would only be for one distribution (fedora) if I understand correctly. I don't think that both (sysext and addon as an option) would do any harm. But of course I don't want to impose anything here

**Comment by travier** [comment]
Agree, those sysexts would be for Fedora only. PRs welcomed for addons.

**Comment by TheRsKing** [comment]
unfortunately the whole thing is still a good bit too complex for my knowledge, anyone who sees that and is interested is welcome to take a look at building such an addon ;)

**Comment by dnkmmr69420** [comment]
> Agree, those sysexts would be for Fedora only. PRs welcomed for addons.

and I am also on an atomic system

**Comment by tulilirockz** [comment]
Hello everyone! The QEMU extension should work just fine now, could someone test it out a bit and report success/failures? Added instructions in https://github.com/flathub/org.virt_manager.virt-manager/pull/13#issuecomment-2602290347
